packages/vscode-extension/src/treeview/account/azureNode.ts

```diff
@@ -34,7 +34,7 @@ export class AzureAccountNode extends DynamicNode {
       }
     }
     this.contextValue = "signedinAzure";
-    this.eventEmitter.fire(undefined);
+    this.eventEmitter.fire(this);
```

Comment:
Only update azure node instead of the whole tree. This can fasten the UI update significantly: **before this change, it could take 10s+ to refresh the whole tree; after this change, the label will get updated in 1s.**